modules/local/dada2_taxonomy.nf

```diff
@@ -26,6 +26,9 @@ process DADA2_TAXONOMY {
     set.seed(100) # Initialize random number generator for reproducibility
 
     seq <- getSequences(\"$fasta\", collapse = TRUE, silence = FALSE)
+    if ( length(seq) < 1 ) {
+        stop("assignTaxonomy: No sequences found in fasta file $fasta")
+    }
```

Comment:
Is that a thing? If this could be due to the empty output of itsx, than I'd rather make process itsx fail instead of this one (which is downstream).

Reply:
For me it has only been a problem when ITSx doesn't find any ITS regions, so I guess it's reasonable to move the error to the itsx module. Will do that.